docs/cri/config.md

```diff
@@ -471,17 +471,23 @@ version = 2
 ## Registry Configuration
 
 Here is a simple example for a default registry hosts configuration. Set
-`config_path = "/etc/containerd/certs.d"` in your config.toml for containerd.
+`config_path = "/etc/containerd/hosts.d"` in your config.toml for containerd.
 Make a directory tree at the config path that includes `docker.io` as a directory
 representing the host namespace to be configured. Then add a `hosts.toml` file
 in the `docker.io` to configure the host namespace. It should look like this:
 ```
-$ tree /etc/containerd/certs.d
-/etc/containerd/certs.d
-└── docker.io
-    └── hosts.toml
 
-$ cat /etc/containerd/certs.d/docker.io/hosts.toml
+$ tree /etc/containerd
+/etc/containerd/
+├── certs.d
```

Comment:
See my comment on the linked to issue, we shouldn't include a directory named `certs.d` here.

Reply:
> See my comment on the linked to issue, we shouldn't include a directory named `certs.d` here.

I have updated the docs